mods/ra/maps/soviet-08b/soviet08b.lua

```diff
@@ -0,0 +1,176 @@
+--[[
+   Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
+alliedScouts = { Jeep1, Jeep2, Rifle1, Rifle2, Rifle3, Rifle4 }
+
+ussrReinforcements = 
+{
+	one =
+	{
+		actors = { "e1", "e1", "e1", "e1", "e1" },
+		entryPath = { SouthEntry.Location, Unload1.Location + CVec.New(1, 0) },
+		exitPath = { SouthEntry.Location },
+	},
+	two =
+	{
+		actors = { "e4", "e4", "e1", "e1", "e1" },
+		entryPath = { SouthEntry.Location, Unload2.Location + CVec.New(0, 1) },
+		exitPath = { SouthEntry.Location }
+	},
+	mammoth =
+	{
+		actors = { "4tnk" },
+		entryPath = { SouthEntry.Location, Unload1.Location + CVec.New(0, 1) },
+		exitPath = { SouthEntry.Location }
+	}
+}
+
+Obj2ActorTriggerActivator = { Church, Civ1, Civ2, Civ3, Civ4, Civ5, Civ6, Civ7, Civ8 }
+
+ActivateAIDelay = DateTime.Seconds(45)
+
+FirstReinforcementTrigger = function()
+	Trigger.AfterDelay(DateTime.Seconds(30), function()
+		Media.PlaySpeechNotification(ussr, "ReinforcementsArrived")
+		local reinforcement = ussrReinforcements.one
+		Reinforcements.ReinforceWithTransport(ussr, "lst.reinforcement", reinforcement.actors, reinforcement.entryPath, reinforcement.exitPath)
+	end)
+end
+
+SecondReinforcementTrigger = function()
+	Trigger.AfterDelay(DateTime.Seconds(60), function()
+		Media.PlaySpeechNotification(ussr, "ReinforcementsArrived")
+		local reinforcement = ussrReinforcements.two
+		Reinforcements.ReinforceWithTransport(ussr, "lst.reinforcement", reinforcement.actors, reinforcement.entryPath, reinforcement.exitPath)
+	end)
+end
+
+AddParadropReinforcementTrigger = function()
+	Trigger.AfterDelay(DateTime.Seconds(90), function()
+		Media.PlaySpeechNotification(ussr, "ReinforcementsArrived")
+		scripteddrop.TargetParatroopers(ScriptedParadrop.CenterPosition, Angle.New(0))
+	end)
+end
+
+ChurchAmbushTrigger = function()
+	if not AmbushSwitch then
+		local hiding = Reinforcements.Reinforce(germany, { 'e1', 'e1', 'e1', 'e1', 'e1', 'e1', 'e1', 'e3', 'e3', 'e3' }, { ChurchAmbush.Location, AmbushMove.Location }, 0)
+		Utils.Do(hiding, function(actor)
+			IdleHunt(actor)
+		end)
+	end
+	AmbushSwitch = true
+end
+
+Trigger.OnKilled(Church, function()
+	Actor.Create("moneycrate", true, { Owner = ussr, Location = ChurchAmbush.Location })
```

Comment:
I forgot if we disabled civilians spawning from the church already. Did we?

Reply:
We haven't. So far I haven't observed a civ picking up this crate. I think that has to do with creating the actor here instead of spawning one on death with a rules override.

Reply:
I guess you were just lucky since I don't see the spawners disabled here. Can you apply a fix like #18318?

Reply:
Done